maintenance/datastore-migration.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Migrate Calico Data from an etcdv3 Datastore to a Kubernetes Datastore
+description: Migrate your cluster from using an etcdv3 datastore to a Kubernetes datastore.
+canonical_url: '/maintenance/datastore-migration'
+---
+
+## Big Picture
+
+Switch the Calico datastore from etcdv3 to Kubernetes.
+
+## Value
+
+In order to utilize all of Calico's newest features, Calico backed by
+the kubernetes datastore is required. Migrating your data allows users
+with an existing cluster set up with an etcdv3 datastore to seamlessly
+transition their cluster to a Kubernetes datastore and take advantage
+of new Calico functionality. For more on the advantages of using a
+Kubernetes datastore over an etcd datastore, see this section of the
+[Calico Datastore]({{ site.baseurl }}/getting-started/kubernetes/hardway/the-calico-datastore#using-kubernetes-as-the-datastore)
+documentation.
+
+## Before you begin...
+
+[calicoctl]({{ site.baseurl }}/getting-started/clis/calicoctl/install) must be installed
```

Comment:
[calicoctl must be installed and configured]({{ site.baseurl }}/getting-started/clis/calicoctl/install)